apiary.apib

```diff
@@ -156,6 +174,10 @@ Check if order can be placed.
 
 ## Customer profile & management [/customer]
 
+!! tohle vůbec nechápu. To je novinka řešená v rámci této druhé části? Buď bude tenhle dokument pro celé API (což teď není) nebo jenom pro novou funkcionalitu a teprve nakonec to sloučíme. Preferuji druhou možnost, bude teď přehlednější. 
+!! Hlavně tady chybí to nejdůležitější, login zákazníka kartou. Data connector vám předá token přečtené karty a vy se s ním přihlásíte. Takže budou dva loginy, jménem a heslem nebo tokenem karty. 
+
+
```

Comment:
Tato část je zkopírovaná z předchozí části projektu, obsahuje pouze změny, které reflektují potřeby pro druhou část. Dokument má být pro celé API, pokud něco chybí je to chyba. Login kartou se provádí dle popisu ### Login [POST /customer/login]

Reply:
Co přesně znamená "Obsahuje pouze změny" vs "Dokument má být pro celé API"?
Každopádně:
- Potřebujeme někde vidět, k jakým změnám proti aktuálně vytvořenému API došlo, aby jsme mohli vytvořit zadání na potřebné změny. 
- Dokument rozhodně celé API neobsahuje, myslím, že tady chybí minimálně receptury. 
Co se týče loginu, tak máte na mysli funkci popsanou "Verifies combination of customer email and password by authorization token."?!

Reply:
doplnil jsem identifikator zda se jedná o novou metodu či změnu, nejasné popisky upravil